wrapper/wrapperText.tsx

```diff
@@ -0,0 +1,13 @@
+import "./wrapper.css";
+
+function WrapperText(props: string) {
```

Comment:
го пропсы сделаем объектом
```suggestion
function WrapperText(props: {content: string}) {
```

Reply:
ну и в использовании тогда будет props.content